.pylintrc

```diff
@@ -56,3 +56,5 @@ enable=
 # Ignore long lines containing URLs or pylint or mypy directives.
 ignore-long-lines=^(.*#\w*pylint: disable.*|.*# type: ignore.*|\s*(# )?<?https?://\S+>?)$
 
+# Ignore python scripts that generate example notebooks
+ignore-patterns=Quantinuum_.*.py
```

Comment:
Can we make this match anything in the `examples/python` directory, instead of the filename pattern? Seems more robust that way.

Reply:
I think this is only supported from pylint>=2.9 (https://github.com/PyCQA/pylint/releases/tag/v2.9.0). I'll upgrade our version and see if nothing else breaks :)